docs/Breaking API Changes.md

```diff
@@ -46,3 +46,10 @@ The change affects compatibility in two ways:
 - DateTimeConstant(-1) will still count when we check that you don’t specify two default values. The compiler will produce an error, instead of succeeding (and producing IL with two attributes).
 
 PR: https://github.com/dotnet/roslyn/pull/11536
+
+# Version 4.1.0
+
+### Can no longer inherit from CompletionService and CompletionServiceWithProviders
```

Comment:
CompletionServiceWithProviders is still protected in this PR since it's used by TS and F#. The plan is to migrate TS and F# to external access API once this PR is merged and follow up with making this ctor internal as well.

Reply:
@333fred do you know if OmniSharp does this?

Reply:
(ideally not, but we should understand if it does, and communicate how to deal with any fallout here if so).

Reply:
We do not inherit from `CompletionService`

Reply:
It's probably also worth making sure that sharplab doesn't do this; @ashmind?

Reply:
Thanks for the ping. MirrorSharp does not inherit from `CompletionService` and uses `GetService` to get it:
https://github.com/ashmind/mirrorsharp/blob/4644ddc79668b7f27ea9660082aad7ba7d2e95f2/Common/Internal/Roslyn/RoslynSession.cs#L60

Reply:
Looks like the F# editor bits in VS Mac inherit from `CompletionServiceWithProviders`:
https://github.com/xamarin/visualfsharp/blob/fsharp-editor/vsintegration/src/FSharp.Editor/Completion/CompletionService.fs#L33
https://github.com/xamarin/visualfsharp/blob/fsharp-editor/vsintegration/src/FSharp.Editor/VSMac/CompletionService.fs#L33

Everything else in VS Mac uses `GetService`.

Reply:
@davidwengier I am aware of F#. I guess the above is a fork of https://github.com/dotnet/fsharp and will get updated once dotnet/fsharp is updated?

Reply:
I think it was a point-in-time copy, and not something that is automatically updated. I would imagine we could at least have it use the same EA as dotnet/fsharp though. @nosami any thoughts?

I guess VS Macs existing IVT might just make this a moot point anyway

Reply:
@davidwengier what's `EA`? :) 
I don't think the IVT helps here as there is no IVT to FSharp.Editor.

@tmat I can update the fork manually once dotnet/fsharp has been updated.